src/pynwb/file.py

```diff
@@ -384,7 +384,10 @@ def create_epoch(self, **kwargs):
         enclosure versus sleeping between explorations)
         """
         self.__check_epochs()
-        self.epoch_tags.update(kwargs.get('tags', list()))
+        if kwargs['tags'] is None:
+            self.epoch_tags.update(list())
```

Comment:
I apologize in advance for being picky. This line does nothing. You could use fewer lines of code if you only update if kwargs['tags'] is not None

Reply:
Oh ok I didn't know if the empty list was important. Thanks for the improvement!